src/GitPrompt.ps1

```diff
@@ -97,6 +97,8 @@ $global:GitPromptSettings = [pscustomobject]@{
 
     EnableWindowTitle                           = 'posh~git ~ '
 
+    AnsiConsole                                 = $Host.UI.SupportsVirtualTerminal -or ($Env:ConEmuANSI -eq "ON")
```

Comment:
I'm not sure there's a practical difference, is there? If `SupportsVirtualTerminal` isn't set, it's falsey.